backend/infrastructures/apps/load/admin.py

```diff
@@ -1 +1,14 @@
-# Register your models here.
+from django.contrib import admin
+from django.db import models
+
+from .models import Data
+
+
+@admin.register(Data)
+class OutputDataAdmin(admin.ModelAdmin):
+    data = models.JSONField()
```

Comment:
@taralalukasz 
Is there another way of representing JSON on admin panel?
I'm not sure, but it's worth to check documentation about some tips.